<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalHistoryFunc</class>
 <widget class="QWidget" name="MedicalHistoryFunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="searchMedicalHistory">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>SearchMedicalHistory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PatientId">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="MedicalHistoryInfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
